en/integrations/plugin-packs/procedures/operatingsystems-linux-nrpe3.md

```diff
@@ -0,0 +1,279 @@
+---
+id: operatingsystems-linux-nrpe3
+title: Linux NRPE3
+---
+
+| Current version | Status | Date |
+| :-: | :-: | :-: |
+| 3.1.6 | `STABLE` | Jun 06 2020 |
+
+## Overview
+
+NRPE (Nagios Remote Plugin Executor) is a protocol that was designed to remotely run monitoring commands.
+
+This plugin pack relies on 3 components that are listed below:
+
+| Component                 | Type              | Location       | Role                                                               |
+| ------------------------- | ----------------- | -------------- | ------------------------------------------------------------------ |
+| `centreon_nrpe3_plugin`   | Binary executable | Poller         | Sends a command name and arguments associated to it                |
+| `centreon_linux_local.pl` | Perl Script       | Monitored host | Is run locally and returns a status, an output message and metrics |
+| `centreon-nrpe3`          | Service/daemon    | Monitored host | Listens on port 5666 and runs the configured commands on demand    |
+
+## Plugin-Pack assets
+
+### Monitored objects
+
+#### Monitored hosts
+
+* CentOS 7 / RHEL7 Linux servers (to which this procedure applies)
+* Other GNU/Linux distributions, provided that the plugin and the daemon are installed and configured manually)
+
+#### Monitored services
+
+Here is the complete list of service templates aliases provided by this plugin pack. "Linked by default" indicates whether the template is linked to the "out-of-the-box" host template.
+
+| Alias                      | Linked by default |
+| :------------------------- | :---------------: |
+| Cmd-Generic                |                   |
+| Connections-Generic        |                   |
+| Cpu-Detailed               |                   |
+| Cpu                        |         X         |
+| Disk-Generic-Name          |                   |
+| Disk-Global                |                   |
+| Disk-IO-Generic-Name       |                   |
+| Disk-IO-Global             |                   |
+| File-Date-Generic          |                   |
+| File-Size-Generic          |                   |
+| Inodes-Generic-Name        |                   |
+| Inodes-Global              |                   |
+| Is-File-Generic            |                   |
+| Is-Not-File-Generic        |                   |
+| Load                       |         X         |
+| Memory                     |         X         |
+| Packet-Errors-Generic-Name |                   |
+| Packet-Errors-Global       |                   |
+| Process-Generic            |                   |
+| Swap                       |         X         |
+| Traffic-Generic-Name       |                   |
+| Traffic-Global             |                   |
+| Uptime                     |         X         |
```

Comment:
I think it will be hard to maintain such array. It brings relevant informations to the reader but, we should stick to objects and metrics for now.